packages/synapse-react-client/src/lib/utils/hooks/useDetectSSOCode.ts

```diff
@@ -16,12 +16,10 @@ import { TwoFactorAuthErrorResponse } from '../synapseTypes/ErrorResponse'
 import { PROVIDERS } from '../../containers/auth/AuthenticationMethodSelection'
 
 type UseDetectSSOCodeOptions = {
+  onSignInComplete?: () => void
   registerAccountUrl?: string
   onError?: (err: unknown) => void
-  onTwoFactorAuthRequired?: (
-    resp: TwoFactorAuthErrorResponse,
-    callback: () => void,
```

Comment:
This `callback` parameter is unused, I think it was mistakenly thrown in when I originally refactored this.